src/types/onyx/PolicyJoinMember.ts

```diff
@@ -0,0 +1,17 @@
+import type * as OnyxCommon from './OnyxCommon';
+
+type PolicyJoinMember = {
+    /** Role of the user in the policy */
```

Comment:
Hmm "Role of the user" is stored in `policyID`? Should this be updated?

Reply:
Yes, it should. Thanks for noticing this! https://github.com/Expensify/App/pull/41462